src/app/feedback/feedback-view.component.ts

```diff
@@ -47,6 +50,8 @@ export class FeedbackViewComponent implements OnInit, OnDestroy {
         this.setCouchListener(result.docs[0]._id);
       }, error => console.log(error));
     this.user = this.userService.get();
+    this.usersService.usersListener().subscribe(users => { users.map(u => this.users[u.doc.name] = u.fullName || u.doc.name ); });
```

Comment:
`Array.map` should not create side effects (modify properties outside of its scope).  Please refactor.